cassdegrees/templates/list.html

```diff
@@ -0,0 +1,42 @@
+{% extends "base.html" %}
+
+{% block page-title %}Template/Subplan List{% endblock %}
+
+{% block subtitle %}Templates{% endblock %}
+
+{% block content %}
+    <table class="fullwidth tbl-cell-bdr">
```

Comment:
Perhaps http://webpublishing.anu.edu.au/web-style-guide/page-tabs.php would be the way to go here.